spec.bs

```diff
@@ -2944,6 +2984,9 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|config|)</dfn> meth
   1. [=list/For each=] |ig| of the [=user agent=]'s [=interest group set=]:
     1. If |ig|'s [=interest group/ads=] is null or [=list/is empty=], [=iteration/continue=].
     1. Let |owner| be |ig|'s [=interest group/owner=].
+    1. If |config|'s [=auction data config/per buyer config=] [=map/is empty=] or

```

Comment:
This looks like it doesn't send any IGs if there is no perBuyerConfig, and I thought we sent everything then?

Reply:
Good catch. Fixed.